### ISSUE TITLE
fixrule(RPT_Header_HasContent) downgrade to recommendation

### ISSUE THREAD

**Issue by philljenkins** [IBMa/equal-access#1315]
Conversation this morning in ACT working group regarding empty headings. Basically unanimous that empty headings are not a violation of WCAG because they’re not rendered, and ATs now ignore them.

headings, `<h_>` not table headers.

That rule id is actually named wrong. It’s about headings not headers
[RPT_Header_HasContent.ts](https://github.com/IBMa/equal-access/blob/master/accessibility-checker-engine/src/v4/rules/RPT_Header_HasContent.ts)

**Comment by philljenkins** [comment]
Change messages:
            "Pass_0": "~~Rule Passed~~ Heading element has descriptive text",
            "Fail_1": "Heading element has no descriptive content",
            "group": "Heading elements ~~must~~should provide descriptive text"

**Comment by philljenkins** [comment]
ruleID will be changed when all ruleIDs are changed for consistency
RPT_Header_HasContent -to- heading_content_exists

**Comment by philljenkins** [comment]
ACT mapping
the current rule is Proposed, and will probably change
Proposed - Heading has non-empty accessible name - ffd0e9
https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/ffd0e9/proposed/

**Comment by philljenkins** [comment]
@tombrunet also, did the ACT group agree to update the Understanding doc to make this interpretation explicit?
https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels

**Comment by aliunwala** [comment]
From Triage:
Mapping this to WCAG 2.4.6 and do not map to ACT as it is going away from ACT

**Comment by sct-will** [comment]
Gated by build issue.

**Comment by ErickRenteria** [comment]
Validated
